tests/capabilitiesTest.js

```diff
@@ -8,12 +8,52 @@ describe('Main: Currently testing getConfig, getVersion and getCapabilities', fu
   // PACT setup
   const Pact = require('@pact-foundation/pact-web')
   const provider = new Pact.PactWeb()
-  const { setGeneralInteractions, validAuthHeaders, xmlResponseHeaders, ocsMeta } = require('./pactHelper.js')
+  const {
+    capabilitiesGETRequestValidAuth,
+    CORSPreflightRequest,
+    GETRequestToCloudUserEndpoint,
+    validAuthHeaders,
+    xmlResponseHeaders,
+    ocsMeta
+  } = require('./pactHelper.js')
+
+  const createOwncloud = async function () {
+    oc = new OwnCloud({
+      baseUrl: config.owncloudURL,
+      auth: {
+        basic: {
+          username: config.username,
+          password: config.password
+        }
+      }
+    })
+    await oc.login()
+    return oc
+  }
```

Comment:
this is repeated in every file, can we move it to the helper?

Reply:
fixed